integration/helpers/base_test.py

```diff
@@ -351,3 +370,32 @@ def verify_stack(self):
         error = verify_stack_resources(self.expected_resource_path, self.stack_resources)
         if error:
             self.fail(error)
+
+    def verify_response(self, url, expected_status_code):
+        print("Making request to " + url)
+        response = requests.get(url)
+        self.assertEqual(response.status_code, expected_status_code, " must return HTTP " + str(expected_status_code))
+        return response
+
+    def get_parameters(self):
```

Comment:
Maybe rename this to `get_test_template_parameters`.
Add doctrings

Reply:
Or even `get_default_test_template_parameters`

Reply:
The name make more sense, will update it.